### PR TITLE
2422-Refactor submission result relationship

### PR DESCRIPTION
**Pull request by aclar48**

### CODE REVIEW

File: config/initializers/bullet.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  #                 :password => 'bullets_password_for_jabber',
+  #                 :receiver => 'your_account@jabber.org',
+  #                 :show_online_status => true }
+  Bullet.rails_logger = false
+  Bullet.add_footer = true
+  # Bullet.stacktrace_includes = [ 'your_gem', 'your_middleware' ]
+  # Bullet.stacktrace_excludes = [ 'their_gem', 'their_middleware' ]
+  # Bullet.slack = { webhook_url: 'http://some.slack.url', foo: 'bar' }
+  Bullet.unused_eager_loading_enable = false
+end
```
Comment (aclar48) [inline config/initializers/bullet.rb]: hound is wrong here

**Comment by aclar48** [comment]
This is good to be merged pending review

**Comment by david-yz-liu** [comment]
Hi @aclar48, thanks for your work on this. I appreciate your commits to address the Hound comments - if you want to change the .rubocop.yml file to not complain about multiline lambdas, I'm okay with that change.

I did a quick search through the app for remaining uses of the old submission->remark result association, and found a few places where `remark_result_id` is used. Could you please add this as an instance method (or if you like, rewrite the places where it's used)?

**Comment by aclar48** [comment]
remark_result_id method added to submission

**Comment by david-yz-liu** [comment]
Thanks, @aclar48!

**Pull request merged**